src/style.css

```diff
@@ -1 +1,60 @@
 /* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+* {
+  margin: none;
+}
+
+.header {
+  margin-top: 0.5%;
```

Comment:
remove margin-top from here, and reset the default body's margin

<img width="1312" alt="Screenshot 2022-06-16 at 17 33 40" src="https://user-images.githubusercontent.com/57598825/174093799-fbe5957a-385d-432a-b0ef-7b4cbed09cec.png">


```suggestion
```